sdk/web-pubsub/web-pubsub-client/src/webPubSubClient.ts

```diff
@@ -486,11 +532,15 @@ export class WebPubSubClient {
     groupName: string,
     content: JSONTypes | ArrayBuffer,
     dataType: WebPubSubDataType,
-    options?: SendToGroupOptions
+    options?: SendToGroupOptionsInternal
   ): Promise<WebPubSubResult> {
     const fireAndForget = options?.fireAndForget ?? false;
     const noEcho = options?.noEcho ?? false;
     if (!fireAndForget) {
+      let ackId = undefined;
+      if (options && "ackId" in options) {
```

Comment:
Could we use `Object.hasOwn` here instead of the `in` check?

Reply:
`Object.hasOwn` start from node 16.9.0. Not sure any concern using `in`?

Reply:
We are targeting Node 18+ from now on, so using that should be safe

Reply:
The typescript seems doesn't handle well for `Object.hasOwn`. https://github.com/microsoft/TypeScript/issues/44253  The following code will result in error in compile("error TS2339: Property 'ackId' does not exist on type 'SendEventOptionsInternal").
```
if (options && Object.hasOwn(options, "ackId")) {
     ackId = options.ackId;
}
```
And it needs to target `ES2022`. I'm not sure whether it will introduce new compatibility issue.
Still I don't know any problem with `in`. Seems it's well supported.